### PR TITLE
added the two library storage services

### PR DESCRIPTION
**Pull request by paulstey**
This PR adds the two storage services that the Library provides to the storage finder app.

### CODE REVIEW

File: services/file-storage-and-transfer/index.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ storage_tool_header: |
   and decide which of these services suit your needs.
 
 services:
-  - service: google_drive
+  - service: google_Drive
```
Comment (mcmcgrath13) [inline services/file-storage-and-transfer/index.yml:17]: why capital D?

File: services/file-storage-and-transfer/index.yml [hunk 2]
```diff
@@ -176,6 +194,122 @@ services:
...
+  - service: LabArchives
+    description: |
+      LabArchives is a cloud-based electronic lab notebook that can be used by researchers, instructors, and 
+      students for input and organization of laboratory data, information sharing, and collaboration, and for 
+      saving historical versions of files. It is appropriate for use in a wide variety of laboratories, including 
+      biological sciences, chemistry and physical sciences, and engineering, among others.
+
+      LabArchives at Brown provides unlimited storage space. The current size limit per file is 4GB.
+
+      LabArchives at Brown is not approved for storing files containing Personally Identifiable Health
```
Comment (mcmcgrath13) [inline services/file-storage-and-transfer/index.yml:268]: PIHI? shouldn't it be PHI?

File: services/file-storage-and-transfer/index.yml [hunk 3]
```diff
@@ -301,6 +447,17 @@ questions:
       - answer: Level 3
         category_classes:
           - 3
+  - question: Do you need a DOI for the data?
```
Comment (mcmcgrath13) [inline services/file-storage-and-transfer/index.yml:450]: but there are ways to get DOIs on data other ways - is this the best way to phrase this?

File: services/file-storage-and-transfer/index.yml [hunk 4]
```diff
@@ -272,7 +418,7 @@ services:
         notes:
 
 questions:
-  - question: What is the classification of your data?
+  - question: What is the risk classification of your data?
     information: |
       Brown has classified its information assets into one of four risk-based categories 
       (No Risk, Level 1, Level 2, or Level 3) for the purpose of determining who is allowed
```
[the remark this answers is not in the document]
Comment (paulstey) [reply services/file-storage-and-transfer/index.yml:424]: Thanks, Isa! This actually is already being done; it's in one of the storage app buttons, not in the content.

File: services/file-storage-and-transfer/index.yml (hunk 1 above)
Comment (paulstey) [reply services/file-storage-and-transfer/index.yml]: Done!! Thanks, @mcmcgrath13!!

File: services/file-storage-and-transfer/index.yml [hunk 5]
```diff
@@ -265,8 +265,8 @@ services:
 
       LabArchives at Brown provides unlimited storage space. The current size limit per file is 4GB.
 
-      LabArchives at Brown is not approved for storing files containing Personally Identifiable Health 
-      Information or Brown Restricted Information.
+      LabArchives at Brown is not approved for storing files containing Personally Identifiable Information (PII) or
```
Comment (mcmcgrath13) [inline services/file-storage-and-transfer/index.yml:268]: comma separated list instead? break out that oxford comma perhaps

File: services/file-storage-and-transfer/index.yml [hunk 6]
```diff
@@ -447,7 +447,7 @@ questions:
       - answer: Level 3
         category_classes:
           - 3
-  - question: Do you need a DOI for the data?
+  - question: Would you like a DOI generated for your data?
```
[the remark this answers is not in the document]
Comment (paulstey) [reply services/file-storage-and-transfer/index.yml]: Done! Thanks, @mcmcgrath13!!

**Review by mcmcgrath13 — APPROVED** [review]

**Pull request merged**